filebeat/module/mysql/slowlog/config/slowlog.yml

```diff
@@ -8,4 +8,4 @@ multiline:
   pattern: '^# User@Host: '
   negate: true
   match: after
-exclude_lines: ['^[\/\w\.]+, Version: .* started with:.*']   # Exclude the header
+exclude_lines: ['^[\/\w\.]+, Version: .* started with:.*', '^# Time:.*']   # Exclude the header and time
```

Comment:
Maybe I'm misunderstanding this, but should we be using this new `Time: ` header for `@timestamp instead of ignoring it?

Reply:
Yeah I think there are some magic here that I don't understand. I'm imagining this exclude_lines is only ignoring it when first time parsing `# User@Host`. After that somehow this `Time` still get parsed into `@timestamp` since it is captured in `mysql/slowlog/test/mysql-5.7.22.log-expected.json`.

Reply:
Did not investigate but I'm also a bit confused on why this keeps working. Perhaps the exlcude does not actually work here? Have a look at the pipeline.

Also it would be good to have in the test file below a few more lines to make sure it also works with multiple entries.

Reply:
@andrewvc @ruflin I looked into this more and found out `@timestamp` field is generated from `mysql.slowlog.timestamp` with format `UNIX`. That's why adding `# Time` into the `exclude_lines` and that didn't change the output.